server/integration/versions/pom.xml

```diff
@@ -39,6 +39,7 @@
       <version.org.infinispan>${project.version}</version.org.infinispan>
       <version.org.infinispan.infinispan-management-console>9.0.0.Final</version.org.infinispan.infinispan-management-console>
       <version.org.infinispan.arquillian.container>1.2.0.Beta1</version.org.infinispan.arquillian.container>
+      <version.org.infinispan.server-build.old>8.0.0.Final</version.org.infinispan.server-build.old>
```

Comment:
This version is also in server/integration/testsuite/pom.xml, can we keep it in only one place?